### PR TITLE
Support using a HTTP proxy for CRI-O and Argo

### PR DESCRIPTION
**Pull request by klausenbusk**
Configure Nidhogg to start the lb-proxy service and configure CRI-O and Argo to use it.

### CODE REVIEW

File: templates/boot.sh [hunk 1]
```diff
@@ -50,6 +50,10 @@ case $NODE_TYPE in
               --set yggdrasil.loadbalancer.ipRangeStart=$$LB_IP_RANGE_START \
               --set yggdrasil.loadbalancer.ipRangeStop=$$LB_IP_RANGE_STOP \
               --set yggdrasil.ingress.loadBalancerIP=$$INGRESS_LB_IP_ADDRESS \
+              --set lb-proxy.enabled=true \
```
Comment (jacobcrawford) [inline templates/boot.sh:53]: I dont think we should enable the lb-proxy always. To be backwards compatible while in transition to the sync service setup, we should be able to disable these features if left out of the mukube-configurator configuration.

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Action needed for backward compatibility.

File: docs/config.md [hunk 2]
```diff
@@ -49,6 +51,12 @@ Last IP address in the range to allocate for the load balancer.
 ### INGRESS_LB_IP_ADDRESS
 Load balancer IP to allocate for the ingress.
 
+### HTTP_PROXY_SERVER
```
Comment (jacobcrawford) [inline docs/config.md:54]: We should be able to disable the use of a proxy to be backward compatible right? Default should be disabled.

File: templates/boot.sh (hunk 1 above)
Comment (klausenbusk) [reply templates/boot.sh]: I agree, this was only posted to get some early feedback. The HTTP proxy feature is opt-in now.

File: docs/config.md (hunk 2 above)
Comment (klausenbusk) [reply docs/config.md:57]: See https://github.com/distributed-technologies/mukube-configurator/pull/36#discussion_r747366709

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
I think we need to document how to inject a "values" file for a helm chart. Otherwise, it looks good.

**Review by jacobcrawford — APPROVED** [review]
LGTM

**Pull request merged**